akka-docs/src/main/paradox/typed/actor-lifecycle.md

```diff
@@ -52,6 +52,38 @@ Java
 
 Refer to @ref:[Actors](actors.md#introduction) for a walk-through of the above examples.
 
+### SpawnProtocol
+
+The guardian actor should be responsible for initialization of tasks and create the initial actors of the application,
+but sometimes you might want to spawn new actors from the outside of the guardian actor. For example creating an actor
+that handles a HTTP request.
+
+That is not difficult to implement in your behavior, but since this is a common pattern there is a predefined
+message protocol and implementation of a behavior for this. It can be used as the guardian actor of the `ActorSystem`,
```

Comment:
again, I think we don't have to mention the combined with setup part, clever people will figure that out, not so clever people should maybe not do it to start with ;)